modules/platforms/dotnet/Apache.Ignite.Core/Cache/Configuration/CacheConfiguration.cs

```diff
@@ -448,6 +453,16 @@ internal void Write(BinaryWriter writer, ClientProtocolVersion srvVer)
             ExpiryPolicySerializer.WritePolicyFactory(writer, ExpiryPolicyFactory);
 
             writer.WriteCollectionRaw(KeyConfiguration);
+            
+            if (PlatformNearConfiguration != null)
```

Comment:
For some reasons, this config seems to not having affect when creating cache with Ignite.CreateCache(cfg), but it works on Ignite.CreateCache(cfg, nearCfg)

Reply:
@ashapkin can you please provide a full reproducer?